client/v2/client.go

```diff
@@ -426,25 +425,26 @@ func (uc *udpclient) Query(q Query) (*Response, error) {
 // Query sends a command to the server and returns the Response
 func (c *client) Query(q Query) (*Response, error) {
 	u := c.url
-
 	u.Path = "query"
-	values := u.Query()
-	values.Set("q", q.Command)
-	values.Set("db", q.Database)
-	if q.Precision != "" {
-		values.Set("epoch", q.Precision)
-	}
-	u.RawQuery = values.Encode()
 
 	req, err := http.NewRequest("GET", u.String(), nil)
 	if err != nil {
 		return nil, err
 	}
+	req.Header.Set("Content-Type", "")
 	req.Header.Set("User-Agent", c.useragent)
 	if c.username != "" {
 		req.SetBasicAuth(c.username, c.password)
 	}
 
+	params := req.URL.Query()
```

Comment:
we get a new `params` object from the specific request that we are making.

before, we were directly modifying `values` of the structs `URL` object